rita_client/src/heartbeat/mod.rs

```diff
@@ -135,28 +177,43 @@ pub fn send_udp_heartbeat() {
         }
     };
 
-    let res = dns_request.then(move |res| {
+    let _res: std::result::Result<(), RitaClientError> = {
```

Comment:
I think I've commented this before but this block can be removed, we don't need a closure here now that it's not a future. Is it giving you trouble?

Reply:
removed